app/helpers/authorization.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Sinatra
+  module Authorization
+    def unauthorized!
+      flash[:danger] = "You're not authorized. Please Log In."
+      redirect '/'
+    end
+
+    def authorized?
+      session[:email].nil? ? false : true
```

Comment:
no need for the ternary. `nil?` is returns a boolean. Actually, all methods with a question mark return a boolen eg `empty?`, `any?`, `even?`, `include?` etc